src/main/java/org/plumelib/util/StringsPlume.java

```diff
@@ -422,6 +425,8 @@ public static String escapeNonASCII(String orig) {
   private static String escapeNonASCII(char c) {
     if (c == '"') {
       return "\\\"";
+    } else if (c == '\''){
```

Comment:
Please use spacing that is consistent with the surrounding code: add a space within "){".  (I fixed this.)